Fixed in 2.2.1 commit 907426b00bdcd69d9a56ac1870990e8ae8c6fe9f
use rpcbind.target instead of .socket (as the upstream commit uses)
because the future versions of nfs-server.service activate the socket
while ours activate the service.

diff --git a/systemd/nfs-mountd.service b/systemd/nfs-mountd.service
index d908afe..587112b 100644
--- a/systemd/nfs-mountd.service
+++ b/systemd/nfs-mountd.service
@@ -3,6 +3,7 @@ Description=NFS Mount Daemon
 Requires=proc-fs-nfsd.mount
 After=proc-fs-nfsd.mount
 After=network.target
+After=rpcbind.target
 BindsTo=nfs-server.service
 
 Wants=nfs-config.service

